middleware/token.middleware.js

```diff
@@ -0,0 +1,37 @@
+import { checkCookie } from "../utils/checkCookie.js";
+import {
+  isAccessValidHandle,
+  isRefreshValidHandle,
+} from "../utils/tokenCheck.js";
+import { refreshToken, token } from "../utils/tokenGenerator.js";
+import {
+  accessTokenOptions,
+  refreshTokenOptions,
+} from "../constants/constants.js";
+
+export const checkToken = async (req, res, next) => {
+  try {
+    const [accessTokenFromCookie, refreshTokenFromCookie] = checkCookie(
+      req.headers["cookie"],
+    );
+    if (!accessTokenFromCookie && !refreshTokenFromCookie) {
```

Comment:
!accessTokenFromCookie || !refreshTokenFromCookie

Reply:
just move this checks to  isAccessValidHandle and isRefreshValidHandle